Automattic/sensei
# Adapt Sensei Pro Ad on Sensei Home

Fixes #5632

### Changes proposed in this Pull Request

* Adjust the Sensei Pro Ad on Sensei Home so it's similar to the design;

### Testing instructions

On a WP installation with this branch of the Sensei LMS plugin installed, and without Sensei Pro installed, visit Sensei Home and verify:

1. If the styling looks OK compared to the original design;
2. If the "Get Sensei Pro" link open a new tab on the checkout page with one Sensei Pro license added;
3. If the "Learn More" link open a new tab on to the Sensei Pro page on senseilms.com;

### Screenshot / Video

![Sensei Pro Ad](https://user-images.githubusercontent.com/529864/197090099-209f0722-9c58-4ab3-8393-112483e37b70.png)
